### ISSUE TITLE
Docker

### ISSUE THREAD

**Issue by davehun** [wardle/rsterminology#6]
Hi Mark I have added a docker fork in my branch as I wanted to have a play without a full stack.
If you think it would be useful I can do a PR no problem if not

**Comment by wardle** [comment]
Yes definitely - I have it deployed currently on a Windows Server VM within the NHS - but docker would make deployment trivial. It would need a link to a PostgreSQL image and a smallish write directory to store the index.

**Comment by wardle** [comment]
I think it is better to do the import of the SNOMED-CT data dynamically rather as a function of the build of docker image. As such, I'm re-opening this issue and will add a REST endpoint to pass in a file/URL to import arbitrary SNOMED versions. Then, I'll rewrite the docker image to simply create a runnable jar essentially.

**Comment by wardle** [comment]
Actually, thinking about this more, this is a terrible idea from me. Instead, to make docker usage easier, much better to have a barebones docker image for the web service and simply do the import manually from command-line to the docker image.
